scripts/install-worker.sh

```diff
@@ -119,21 +119,26 @@ sudo mv $TEMPLATE_DIR/iptables-restore.service /etc/eks/iptables-restore.service
 ### awscli #####################################################
 ################################################################################
 
-if [[ "$BINARY_BUCKET_REGION" != "us-iso-east-1" && "$BINARY_BUCKET_REGION" != "us-isob-east-1" ]]; then
-  # https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html
-  echo "Installing awscli v2 bundle"
-  AWSCLI_DIR=$(mktemp -d)
-  curl \
-    --silent \
-    --show-error \
-    --retry 10 \
-    --retry-delay 1 \
-    -L "https://awscli.amazonaws.com/awscli-exe-linux-${MACHINE}.zip" -o "${AWSCLI_DIR}/awscliv2.zip"
-  unzip -q "${AWSCLI_DIR}/awscliv2.zip" -d ${AWSCLI_DIR}
-  sudo "${AWSCLI_DIR}/aws/install" --bin-dir /bin/
+if command -v aws &> /dev/null
+then
+    echo "AWS CLI is already installed."
```

Comment:
We would need to add assertions on the CLI version, but that's kind of a pain with the v1/v2 divide. For example `1.25.72`/`2.7.32` is needed for local clusters on Outpost. We also need `describe-addons-versions`. I'd probably just go with what you have in #1205 to always install the latest v2.